.circleci/config.yml

```diff
@@ -39,12 +39,16 @@ jobs:
           command: make tests:prepare
 
       - run:
-          name: Run unit (integration) tests
-          command: make tests:run unit
+          name: Run basic group tests
+          command: make tests:run -- -g basic
 
       - run:
-          name: Run API tests
-          command: make tests:run api
+          name: Install additional modules
+          command: make add:additional-modules
+
+      - run:
+          name: Run basic and additional groups tests
+          command: make tests:run -- -g basic -g additional
```

Comment:
Yep, good idea; we definitely need to run both groups after enabling additional modules.

Reply:
A second thought - we can put all the basic tests into the `additional` group too - not sure what approach is better :)

Reply:
I think we should not add all basic tests to additional group, becouse it is Additional tests only. Probably we can create group All_tests or smth like this. (but if we will not have other groups we will can used  make tests:run )